### PR TITLE
CLDC-2554 Update back to all logs link

### PR DESCRIPTION
**Pull request by kosiakkatrina**
Back to all logs button kept on redirecting back to the same page. It's now clearing all the filters instead.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_07_25_081029) do
+ActiveRecord::Schema[7.0].define(version: 2023_07_19_150610) do
```
Comment (natdeanlewissoftwire) [inline db/schema.rb:13]: why is this included?

**Review by natdeanlewissoftwire — APPROVED** [review]
looks good, not sure why schema is included but approving pending that :)

File: db/schema.rb (hunk 1 above)
Comment (kosiakkatrina) [reply db/schema.rb:13]: I don't think there is a 20230725081029 migration, the latest one in the files seems to be 20230719150610, so I'm not sure why 2023_07_25_081029 was committed in the first place. Lmk if I'm missing something tho
